newm-server/src/main/kotlin/io/newm/server/features/scheduler/OutletReleaseStatusJob.kt

```diff
@@ -0,0 +1,76 @@
+package io.newm.server.features.scheduler
+
+import io.newm.server.features.release.repo.OutletReleaseRepository
+import io.newm.server.features.song.model.MintingStatus
+import io.newm.server.features.song.repo.SongRepository
+import io.newm.shared.koin.inject
+import io.newm.shared.ktx.info
+import io.newm.shared.ktx.toUUID
+import kotlinx.coroutines.runBlocking
+import org.koin.core.parameter.parametersOf
+import org.quartz.DisallowConcurrentExecution
+import org.quartz.Job
+import org.quartz.JobExecutionContext
+import org.quartz.JobKey
+import org.slf4j.Logger
+import java.time.LocalDateTime
+import java.time.ZoneOffset
+
+@DisallowConcurrentExecution
+class OutletReleaseStatusJob : Job {
+    private val log: Logger by inject { parametersOf(javaClass.simpleName) }
+    private val songRepository: SongRepository by inject()
+    private val outletReleaseRepository: OutletReleaseRepository by inject()
+
+    override fun execute(context: JobExecutionContext) {
+        log.info {
+            "OutletReleaseStatusJob key: ${context.jobDetail.key.name} executed at ${
+                LocalDateTime.ofInstant(
+                    context.fireTime.toInstant(),
+                    ZoneOffset.UTC
+                )
+            }"
+        }
+        runBlocking {
+            val songId = context.mergedJobDataMap.getString("songId").toUUID()
+            try {
+                if (outletReleaseRepository.isSongReleased(songId)) {
```

Comment:
In EvearaReleaseStatusJob, we have a `forceDistributed` flag on the song that is set when we're in the staging environment so we can pretend the song was distributed. We will need a similar flag here for `forceReleased` since we won't actually be releasing anything to spotify in the staging environment.